### ISSUE TITLE
Install failed on windows 10

### ISSUE THREAD

**Issue by araraloren** [tony-o/perl6-pluggable#9]
command

    zef install Pluggable

```
===> Searching for: Pluggable
===> Searching for missing dependencies: Test::META
===> Searching for missing dependencies: META6
===> Searching for missing dependencies: JSON::Class
===> Searching for missing dependencies: JSON::Marshal, JSON::Unmarshal
===> Searching for missing dependencies: JSON::Name
===> Testing: JSON::Name:ver('0.0.2'):auth('github:jonathanstowe')
===> Testing [OK] for JSON::Name:ver('0.0.2'):auth('github:jonathanstowe')
===> Testing: JSON::Marshal:ver('0.0.13'):auth('github:jonathanstowe')
===> Testing [OK] for JSON::Marshal:ver('0.0.13'):auth('github:jonathanstowe')
===> Testing: JSON::Unmarshal:ver('0.08')
===> Testing [OK] for JSON::Unmarshal:ver('0.08')
===> Testing: JSON::Class:ver('0.0.9'):auth('github:jonathanstowe')
===> Testing [OK] for JSON::Class:ver('0.0.9'):auth('github:jonathanstowe')
===> Testing: META6:ver('0.0.16'):auth('github:jonathanstowe')
===> Testing [OK] for META6:ver('0.0.16'):auth('github:jonathanstowe')
===> Testing: Test::META:ver('0.0.10'):auth('github:jonathanstowe')
# the following may make some diagnostics from the module itself
===> Testing [OK] for Test::META:ver('0.0.10'):auth('github:jonathanstowe')
===> Testing: Pluggable:ver('0.2'):auth('github:tony-o')

# Failed test at t\01-oop.t line 18

# Failed test at t\01-oop.t line 34
# Looks like you planned 4 tests, but ran 2
# Looks like you failed 2 tests of 2

# Failed test at t\03-proc.t line 15
# Looks like you failed 1 test of 1
===> Testing [FAIL]: Pluggable:ver('0.2'):auth('github:tony-o')
Aborting due to test failure: Pluggable:ver('0.2'):auth('github:tony-o') (use --force-test to override)
  in code  at C:\rakudo\share\perl6\site\sources\1C5B2A3EA861108A89345CD9B974CE9BF1FEE32F (Zef::Client) line 375
  in method test at C:\rakudo\share\perl6\site\sources\1C5B2A3EA861108A89345CD9B974CE9BF1FEE32F (Zef::Client) line 353
  in code  at C:\rakudo\share\perl6\site\sources\1C5B2A3EA861108A89345CD9B974CE9BF1FEE32F (Zef::Client) line 530
  in sub  at C:\rakudo\share\perl6\site\sources\1C5B2A3EA861108A89345CD9B974CE9BF1FEE32F (Zef::Client) line 527
  in method install at C:\rakudo\share\perl6\site\sources\1C5B2A3EA861108A89345CD9B974CE9BF1FEE32F (Zef::Client) line 633
  in sub MAIN at C:\rakudo\share\perl6\site\sources\8CF719A0E6E0BBDC883C69273F5806F53CE89B08 (Zef::CLI) line 152
  in block <unit> at C:\rakudo\share\perl6\site\resources\31793EAD9570EB4504E3FFB7B7A8A54BCD5B0D99 line 1
  in sub MAIN at C:\rakudo\share\perl6\site\bin\zef line 2
  in block <unit> at C:\rakudo\share\perl6\site\bin\zef line 2
```

**Comment by tony-o** [comment]
@araraloren - this should be corrected with the push from robert, are you able to retest?

**Comment by araraloren** [comment]
@tony-o Still not work.
The error log:
```
C:\Users>zef install Pluggable
===> Searching for: Pluggable
===> Testing: Pluggable:ver<0.3>:auth<github:tony-o>
# Failed test at t\01-oop.t line 18
# Failed test at t\01-oop.t line 34
# Looks like you planned 4 tests, but ran 2
# Looks like you failed 2 tests of 2
# Failed test at t\03-proc.t line 15
# Looks like you failed 1 test of 1
===> Testing [FAIL]: Pluggable:ver<0.3>:auth<github:tony-o>
Aborting due to test failure: Pluggable:ver<0.3>:auth<github:tony-o> (use --force-test to override)
```

**Comment by holli-holzer** [comment]
This error happens because you don't take the Windows path separator into account in your code.
Add

    @files = map(-> $s { $s.subst(/\\/, '::', :g) }, @files);

to your code (~ Line 125) and the bug is fixed.

**Comment by tony-o** [comment]
@holli-holzer sorry, this is on my radar - i need to make a windows vm to test and haven't gotten to it yet

**Comment by tony-o** [comment]
This is fixed in fc39c70
